core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -179,31 +179,31 @@ class GroupCoordinator(val brokerId: Int,
 
         if (group.hasStaticMember(groupInstanceId)) {
           val oldMemberId = group.getStaticMemberId(groupInstanceId)
+          info(s"Static member $groupInstanceId with unknown member id rejoins, assigning new member id $newMemberId, while " +
+            s"old member $oldMemberId will be removed. It group is currently stable, no rebalance will be triggered.")
 
-          if (group.is(Stable)) {
-            info(s"Static member $groupInstanceId with unknown member id rejoins, assigning new member id $newMemberId, while " +
-              s"old member $oldMemberId will be removed. No rebalance will be triggered.")
+          val oldMember = group.replaceGroupInstance(oldMemberId, newMemberId, groupInstanceId)
```

Comment:
The use of `oldMember` here is confusing as it refers to the member with the updated memberId. Maybe it could be `activeMember` or just `member`?

Reply:
`member` LGTM